testSrc/io/flutter/analytics/AnalyticsTest.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright 2016 The Chromium Authors. All rights reserved.
+ * Use of this source code is governed by a BSD-style license that can be
+ * found in the LICENSE file.
+ */
+package io.flutter.analytics;
+
+import junit.framework.TestCase;
+import org.junit.Assert;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+public class AnalyticsTest extends TestCase {
```

Comment:
Still JUnit3 (on all the tests I believe). I have no feeling one way or the other (3 / 4), but we should make all the tests consistent if we decide to switch over.